plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerIntegrationTest.java

```diff
@@ -191,4 +195,42 @@ public void allowsLatestTopLevelDependency() throws IOException {
         pluginManager.findPluginsAndDependencies(requestedPlugins);
         assertThatNoException();
     }
+
+    //TODO: Enable as auto-test once it can run without massive traffic overhead
```

Comment:
is this a problem because you used workflow-job? could you use a small simple plugin instead and it would be ok?

Reply:
Workflow-job also emulates dependency updates, so it is useful for deeper tests. Will create a simple fork